### PR TITLE
Rebased Protein LM Tutorial w. Elisa

### PR DESCRIPTION
**Pull request by dhuvik**
## Description

Fix #(issue)

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [ ] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
  - [ ] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: examples/tutorials/ProteinLM_Tutorial0.ipynb [hunk 1]
```diff
@@ -0,0 +1,2853 @@
+{
```
Comment (rbharath) [inline examples/tutorials/ProteinLM_Tutorial0.ipynb:1]: <p>Missing some of the standard deepchem tutorial headers.</p><p> </p><p>Draw a connection to Chemberta and other LM uses in deepchem ecosystem as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (rbharath) [inline examples/tutorials/ProteinLM_Tutorial0.ipynb:1]: <p>Avoid first person "I" and make language a little more formal.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Review by rbharath — COMMENTED** [review]
Small changes; also maybe want to add this to the protein ML section of tutorials.

File: examples/tutorials/ProteinLM_Tutorial0.ipynb (hunk 1 above)
Comment (dhuvik) [reply examples/tutorials/ProteinLM_Tutorial0.ipynb:1]: <p>Removed the informal language and the first person I from the 3blue1brown tutorial and elsewhere. Also made it more timeless as a standalone piece.</p>

**Pull request merged**